features/networking/service.feature

```diff
@@ -384,3 +384,41 @@ Feature: Service related networking scenarios
       | Hello-OpenShift-1 http-8080 |
 
 
+  # @author huirwang@redhat.com
+  # @case_id OCP-11645
+  Scenario: Create loadbalancer service
+    Given I have a project
+    When I run the :create client command with:
+      | f | https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/networking/ping_for_pod_containerPort.json |
+    Then the step should succeed
+
+    # Create loadbalancer service
+    When I run the :create_service_loadbalancer client command with:
+      | name | hello-pod |
+      | tcp  | 5678:8080       |
```

Comment:
Bad indention.

Reply:
fix it, thanks!

Reply:
More cleaner if we use
```
  | name | hello-pod |
```
compared with,
```
  | name | hello-pod       |
```

Reply:
Okay, fix it, thanks!